include/spirv/unified1/spirv.core.grammar.json

```
@@ -6644,6 +6644,211 @@
       ],
       "version" : "None"
     },
+    {
+        "opname" : "OpRayQueryGetIntersectionSpherePositionNV",
+        "class" : "Reserved",
+        "opcode" : 5427,
+        "operands" : [
+            { "kind" : "IdResultType" },
+            { "kind" : "IdResult" },
+            {
+                "kind" : "IdRef",
+                "name" : "'RayQuery'"
+            },
+            {
+                "kind" : "IdRef",
+                "name" : "'Intersection'"
+            }
+        ],
+        "capabilities" : [ "RayQueryKHR" , "RayTracingSpheresGeometryNV" ],
```

I'm not sure why `RayQueryKHR` is in the list, that's not in the extension. Is that because of the operands ?

Same comment for the occurrences below.

```suggestion
        "capabilities" : [ "RayTracingSpheresGeometryNV" ],
```

---

Sorry, didn't updated the comment before submitting. Looking at the changes and the extension, I think there is some confusions about the intent of the capabilities. Multiple capabilities means you need one of them to be able to use the instruction (see https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#Capabilities).  My understanding is you added the extra capability because an operand's type need it, that's not something you can model here.